### PR TITLE
WD-241: Add secrets handling with Silta CLI.

### PR DESCRIPTION
**Pull request by tormi**
### Changes

This PR adds handling of secrets with the help of [Silta CLI](https://github.com/wunderio/silta-cli). This is a replacement for #382.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ Drush alias for the **current** Silta feature branch deployment is `drush @curre
 - `lando varnishadm <commands>` - run [varnishadm](https://varnish-cache.org/docs/6.0/reference/varnishadm.html) commands.
 - `lando xdebug <mode>` - load [Xdebug](https://xdebug.org/) in the selected [mode(s)](https://xdebug.org/docs/all_settings#mode).
 
+#### Secrets handling
+
+Perform encryption and decryption operations by using [Silta CLI](https://github.com/wunderio/silta-cli). Use `silta secrets -h` to see the available commands. Use the `SECRET_KEY` value of the corresponding Silta context environment variable as the password.
```
Comment (misterjoonas) [inline README.md:64]: Should it be mentioned that SECRET_KEY can be found from the password manager (LastPass currently)?

Comment (tormi) [reply README.md:64]: The up-to-date one is always available via the corresponding Silta context. This applies even more when we switch to per-project `SECRET_KEY` approach.

Comment (ArtisKrumins) [reply README.md:64]: Variable name too can be anything as long as it is consistent.
For example
`            - silta/decrypt-files:
                files: silta/silta-prod-secret.yaml
                secret_key_env: MY_SECRET_NAME
`

**Comment by tormi** [comment]
I'll add `circleci/config.yml` changes next.

**Review by misterjoonas — APPROVED** [review]
Looks good to me

**Comment by tormi** [comment]
Decrypting worked as expected in CI:

Screenshot 2024-01-05 at 16 14 00

**Comment by tormi** [comment]
> - TEST_KEY - secret key for tesing purposes.

Typo: tesing > testing

**Pull request merged**